src/test/java/org/folio/dcb/service/TransactionServiceTest.java

```diff
@@ -30,4 +48,29 @@ void createCirculationRequestTest() {
     transactionsService.createCirculationRequest(DCB_TRANSACTION_ID, createDcbTransaction());
     verify(lendingLibraryService).createTransaction(DCB_TRANSACTION_ID, createDcbTransaction());
   }
+
+  @Test
+  void shouldReturnAnyTransactionStatusById(){
+    var transactionIdUnique = UUID.randomUUID().toString();
+    when(transactionRepository.findById(transactionIdUnique))
+      .thenReturn(Optional.ofNullable(TransactionEntity.builder().status(TransactionStatus.StatusEnum.CREATED).build()));
+    when(dcbTransactionMapper.mapToDcbTransaction(any(TransactionEntity.class)))
+      .thenReturn(new DcbTransaction());
```

Comment:
I would suggest to return some valid response (EntityUtils.createTransactionEntity()) instead of new DcbTransaction() and do some other assertions inaddition to non null assertion.

Reply:
done